### DISCUSSION TITLE
Unable to bind to the remote server's port

### DISCUSSION THREAD

**Discussion by john-shipp** [spatie/ray#402] [Q&A]
Hi! Sorry to bother with this but I'm unsure how to fix it. I'm trying to connect to a forge instance via SSH. I was able to get it to connect, then, I restarted my computer (and Ray) and now I'm getting the following error message.

This is probably a simple fix but I can't seem to figure out what to do. :) :) 

Thank you in advance and thank you for such an awesome tool!!!!!

(The port is not in use and nobody else is using ray on that server.)

<img width="426" alt="Screen Shot 2021-03-31 at 12 19 46 PM" src="https://user-images.githubusercontent.com/5049991/113192482-0aa11300-921c-11eb-98de-6437cf612361.png">

**Comment by freekmurze** [comment]
@AlexVanderbist could you take a look?

**Comment by AlexVanderbist** [comment]
I think it might be related to this: https://unix.stackexchange.com/questions/427189/how-to-cleanup-ssh-reverse-tunnel-socket-after-connection-closed

Will look further into it at the start of next week!

**Reply by john-shipp** [reply]
Guessing you're right! I restarted the server and then was able to connect (before I received your reply above). Thank you @AlexVanderbist and @freekmurze !! :)
